how-to/reviewer-guide.md

```diff
@@ -102,6 +102,13 @@ in the review template, please also provide general comments addressing the foll
 
 If you have your own relevant data/problem that you could use the package to solve, work through it with the package. You may find rough edges and use-cases the author didn't think about.
 
+```{important}
+Some maintainers do not ship the tests with their package.
+This means that the dependencies for testing may be missing.
```

Comment:
@cmarmo i'm curious if even if authors ship tests, would they add a tool like pytest as a dependency for the entire package?

Reply:
so there would be a dep section in the pyproject.toml file for dependencies but those might not necessarily get installed with the package install ?

Reply:
If I understand correctly you can define [optional dependencies](https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#declaring-project-metadata) in `pyproject.toml` eg
```
[project.optional-dependencies]
test = ["pytest"]
doc = [
  "matplotlib",
  "click",
]
``` 
then install with eg
```
pip install mypackage[test]
```

Reply:
thank you! i never knew that you could install that way (ie using `[test]` but learned about it today! 

What do you think about us adding a note about how to install with tests using the example that you provide?

i've also made a note to clarify how to run tests in our package guide as well :)

Reply:
> What do you think about us adding a note about how to install with tests using the example that you provide?

I think it could be helpful for packagers to know about the `optional-dependencies` metadata.

The keyword in the install command depends on the choice of the maintainer/packager.
In my example the command to install the doc dependencies is
```
pip install mypackage[doc]
```
but could have been `docs` for example.
If the maintainer adds `optional-dependencies` we might want to check that the install documentation is complete during the review.